test/extended/default_mysql_test.go

```diff
@@ -0,0 +1,49 @@
+// +build default
```

Comment:
Have you tested to be sure that these work how we expect them to when using ginkgo?